app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
```

Comment:
- [ ] Kindly ensure that the devise login page disappears upon successful login
![image](https://user-images.githubusercontent.com/81932478/209351847-4cf4017c-07a0-4c62-a7a2-712609ff84e8.png)

Reply:
This feature has now been implemented! Thanks so much for the review!